tools/scripts/helpers/github.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+import requests
+
+from dotenv import load_dotenv, find_dotenv
+load_dotenv(find_dotenv())
+
+logger = logging.getLogger(__name__)
+
+
+class GitHub:
+    def get_headers(self) -> dict:
+        token = os.environ.get("GITHUB_TOKEN")
+        assert token, "GitHub token is not found."
+        return {"Authorization": "token {}".format(token)}
+
+    def fetch_timeline(self, timeline_url: str) -> list:
```

Comment:
I don't think either of the URLs here actually need a GH token for GET requests, so maybe we could drop that requirement for now?

Reply:
This is mainly to bypass rate limit of 60 requests per hour for unauthenticated users. I guess I could make it optional as on average there are not that many issues for a given bugzilla bug and the likelihood of hitting the limit is low when creating one entry at a time :)